packages/salesforcedx-vscode-soql/src/sfdx.ts

```diff
@@ -17,14 +19,25 @@ export const channelService = ChannelService.getInstance(
 
 export const workspaceContext = WorkspaceContextUtil.getInstance();
 
+function showChannelAndErrorMessage(e: any) {
```

Comment:
You should add telemetry to this error too.

Reply:
@lcampos - I was thinking that having an issue with the connection is just a normal part of sfdx development.  Scratch orgs expire as part of normal development.  Lemme know if you'd still like to see me adding telemetry.  Happy to.

Reply:
You are correct in that this error with expired orgs is expected. However, there is value in knowing the amount of times users run into this. It gives us visibility so we can make product decisions to minimize them running into this or also potentially catching an unrelated error causing spikes on this error scenario.

Reply:
Gotcha.  Sometimes we use telemetry to catch edge cases instead of metrics.  And that's a philosophy flaw.